capDL-tool/capDL-tool.cabal

```diff
@@ -59,3 +59,5 @@ executable parse-capDL
 
   ghc-options:         -O2 -Werror -Wall -fno-warn-name-shadowing
                        -fno-warn-missing-signatures -fno-warn-type-defaults
+                       -fno-warn-incomplete-uni-patterns
+                       -fno-warn-incomplete-record-updates
```

Comment:
Out of curiosity, are these now included in `Wall`?

Reply:
Yes, they failed the build because we stop on all warnings. And they are kind of pretty annoying warnings, the `let` matching feature makes no sense if you can't use it without warning, afaics.

Reply:
Yeah, that was my understanding for why they were previously left out of `Wall`.